### PR TITLE
Use post in stead of get, for massUpdate and massEdit.

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by fernandog** [comment]
Does this fix the `Cannot add all existing shows / mass update due to URL limitation` issue ?

**Comment by p0psicles** [comment]
It does fix the mass update error. But it doesn't fix the add existing show error. I'll look at that separate.

File: static/js/mass-update.js [hunk 1]
```diff
@@ -95,19 +102,18 @@ $(document).ready(function() {
...
-            toDelete: deleteArr.join('|'),
-            toRemove: removeArr.join('|'),
-            toMetadata: metadataArr.join('|')
+        if (updateArr.length + refreshArr.length + renameArr.length + subtitleArr.length + deleteArr.length + removeArr.length + metadataArr.length === 0) {
+            return false;
+        }
+        const url = $('base').attr('href') + 'manage/massUpdate';
+        const params = 'toUpdate=' + updateArr.join('|') + '&toRefresh=' + refreshArr.join('|') + '&toRename=' + renameArr.join('|') + '&toSubtitle=' + subtitleArr.join('|') + '&toDelete=' + deleteArr.join('|') + '&toRemove=' + removeArr.join('|') + '&toMetadata=' + metadataArr.join('|');
+        $.post(url, params, function() {
+            location.reload(true);
```
Comment (OmgImAlexis) [inline static/js/mass-update.js]: Is it the best idea to refresh since we may end up sending the request twice because of the form?

Comment (p0psicles) [inline static/js/mass-update.js]: I don't think you can send the post twice. The reload will simply reload the page.

Comment (labrys) [inline static/js/mass-update.js]: With a nonce you could still refresh and delay the response until action is completed.  A nonce would also prevent a doubled action.

Comment (p0psicles) [reply static/js/mass-update.js]: Let me stop you guys right there. I'm not interested in creating all kind of fancy features in our "old" frontend (like utilizing the websockets for this). I don't want to spend my time on that. If you think that the current PR doesn't hold enough value as is, then i'd say remove it. As I got lot's of things to work on (like bootstrapping the config pages or finishing super's binsearch provider).

Keep in mind, the massUpdate/Edit page in practice is used once by user, and then the user moves on in his/her use flow. Also I still don't see how we can send the request twice **by accident**.

**Pull request merged**